utilities/examine_focus_w_grid.py

```diff
@@ -72,33 +76,28 @@ def variance_of_laplacian(image):
 mv_avg_freq = 5  # Number of frames to perform moving average on
 laplace_array = np.full((num_rows, num_cols, mv_avg_freq), None)
 
-# Initialize camera and grab a reference to the raw camera capture
-camera = PiCamera()
-camera.rotation = 180
-camera.resolution = (640, 480)
-camera.framerate = 30
-rawCapture = PiRGBArray(camera, size=(640, 480))
+# Open a camera for video capturing
+cap = cv2.VideoCapture(0)
 
-# Allow the camera to warmup
-time.sleep(0.1)
 
-# Capture frames from the camera
-for frame in camera.capture_continuous(rawCapture,
-                                       format='rgb',
-                                       use_video_port=True):
-    # Grab the raw NumPy array representing the image
-    color_image = frame.array
+def show_frame():
+    global tkinter_image, mv_avg_count
 
-    tkinter_image = ImageTk.PhotoImage(image=Image.fromarray(color_image))
-    canvas.itemconfig(image, image=tkinter_image)
+    ret, frame = cap.read()
+    frame = cv2.resize(frame, (display_width, display_height))
+    frame = cv2.rotate(frame, cv2.ROTATE_180)
+    color_image = cv2.cvtColor(frame, cv2.COLOR_BGR2RGBA)
+    gray_image = cv2.cvtColor(frame, cv2.COLOR_BGR2GRAY)
 
-    gray_image = cv2.cvtColor(color_image, cv2.COLOR_RGB2GRAY)
+    image_data = Image.fromarray(color_image)
+    tkinter_image = ImageTk.PhotoImage(image=image_data)
+    canvas.itemconfig(image_on_canvas, image=tkinter_image)
 
-    for i in range(num_cols):
-        for j in range(num_rows):
```

Comment:
What is the reason we are flipping the traversal of the rows and columns by reversing the nesting here?

Reply:
I flipped them because it's the method that I was taught for traversing 2D arrays. However, would you prefer that I change it back?

Reply:
Seems a little arbitrary to prefer one traversal over the other.  Was a reason given for always doing it that way?  I could see for specific situations where the data was organized in memory so that the second row comes after the first row etc. and one was touching every element that could make sense but you'd have to be moving through the elements in a way that took advantage of that organization at a low enough level (i.e. with a pointer in C/C++; and then you wouldn't even need a nested loop in many cases) to have a performance benefit. In this situation, I wonder what the advantage would be?

I don't have a huge issue with making the change.  But I do advocate for trying to change as little as possible to realize a feature or address an issue and to keep the PR diff really clear.